packages/frontend/src/components/common/LocalAlertBox.tsx

```diff
@@ -56,19 +57,16 @@ const LocalAlertBoxContainer = styled.div`
     }
 `;
 
-/**
- * Renders request status.
- *
- * @param localAlert {object} request status, can be null in case not completed yet / no outgoing request
- * @param localAlert.success {boolean} true if request was succesful
- * @param localAlert.messageCode {string} localization code of status message to display
- */
-const LocalAlertBox = ({ localAlert, accountId, dots }) => (
+interface Props {
```

Comment:
I think we should maintain the documentation if possible (gives docs inline when using the component props). Generally a comment above each property should suffice, and one above the whole interface. If that doesn't work, then look into `jsdoc` and TypeScript interactions.